scripts/update-gh-pages.sh

```diff
@@ -0,0 +1,209 @@
+#!/bin/bash -e
```

Comment:
this doesn't seem to be releated to PR also I see network-operator.0.3.1.tgz binary which should be removed

Reply:
Yes. It was accidently added  to the PR